tools/sheets/README.md

```diff
@@ -0,0 +1,27 @@
+# Sheets Tools
+
+These are two command-line tools to copy csv files to and from Google Sheets.
+
+## Copy a csv file to Google Sheets
+
+To copy a csv file to Google sheets, run the following command:
+```bash
+./run.sh -m csv2sheet -l <csv_file_path> -s <sheets_url> -w <worksheet_name>
```

Comment:
nit: put these enclosed in `[ ]` following the optional param convention?

Reply:
done